src/preProcessPattern.js

```diff
@@ -60,18 +60,26 @@ export default function preProcessPattern(globalRef, pattern) {
     debug(`determined '${pattern.from}' to be read from '${pattern.absoluteFrom}'`);
 
     return stat(inputFileSystem, pattern.absoluteFrom)
-    .catch(() => {
-        // If from doesn't appear to be a glob, then log a warning
+    .catch((err) => {
         if (isGlob(pattern.from) || pattern.from.indexOf('*') !== -1) {
+            // If it is a glob, then no worries if it does not exist in the file system
             pattern.fromType = 'glob';
             pattern.glob = escape(pattern.context, pattern.from);
+        } else if (!path.isAbsolute(pattern.from)) {
```

Comment:
What about if i want use absolute path?

Reply:
On [line 54](https://github.com/webpack-contrib/copy-webpack-plugin/pull/256/files/cde916a4f46cccc0047732834f2a58d73d9af9fc#diff-a796a48a9ecbc15f8f48dc347832123eR54), we specify that if the given `from` path is already absolute, use it as-is.

Then we try to call `stat` on the file. This edit all exists within the `catch` block, which is only evaluated because the inferred absolute path (which is the same as the path you pass in if it was already absolute) did not actually exist.

So if you passed a valid absolute path in for `pattern.from`, the `stat` function call on line 62 wouldn't throw an error.